tests/tests.ipkg

```diff
@@ -1,5 +1,5 @@
 package runtests
 
-depends = contrib
+depends = contrib, test
```

Comment:
Do we still need the dependency on `contrib`?

Reply:
Oh gosh, this comment is in tests. The answer must be no. Good catch.